### PR TITLE
Adding all file and directory content for package

### PR DESCRIPTION
**Pull request by metalphase**
Adding necessary .toml, readme, requirements files, as well as the contents of the package in src/

### CODE REVIEW

**Comment by Tohuvavohu** [comment]
Why do you need the following pieces

/data/bright_source_cat.fits
/data/F646018360

there may be other extraneous pieces too.

You can look at what you actually need to do the relevant response calculation and tests, and only include that.

**Comment by Tohuvavohu** [comment]
also many of the requirements listed are not needed

File: src/nitrates_response/config.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        "quads": [3],
+        "drm_fname": "drm_-1.100_0.500_.fits",
+        "imx": -1.1,
+        "imy": 0.5,
+        "id": 8,
+    },
+}
+
+
+# email password file
```
Comment (Tohuvavohu) [inline src/nitrates_response/config.py]: Dont need this

**Comment by metalphase** [comment]
@Tohuvavohu  Removed the F64601860 directory and added the test_response.py file that will run using pytest. I also edited the requirements to just include the necessary modules and just cleaned up the repository in general according to your advice

**Review by Tohuvavohu — CHANGES_REQUESTED** [review]
Great job, it's looking really nice!

Two things:
1) If this is set up properly, install should only require 
'pip install .' in the root directory

2) please squash your commits before first merge

**Comment by Tohuvavohu** [comment]
Also, when you've figured that out, install instructions should be in README.md

**Comment by metalphase** [comment]
Looks like the package should install just as you described in 1.  I've included the install instructions into the README.md file. I'm still figuring out how to squash my last commits. Should I be squashing all of my commits into a singular one after your initial commit for the repo?

**Comment by Tohuvavohu** [comment]
Yep!

**Comment by Tohuvavohu** [comment]
also you have an extraneous, empty, file called 'ls' which should be removed

**Comment by metalphase** [comment]
@Tohuvavohu alright, was a nail-biter but was able to squash the commits successfully and now all the commits show up as a singular commit.

**Comment by metalphase** [comment]
Was there any other things that needed to be changed?

**Review by Tohuvavohu — APPROVED** [review]

**Pull request merged**